src/app/app.ts

```diff
@@ -0,0 +1,42 @@
+import {Post} from 'mattermost-redux/types/posts';
+import {AppCall, AppContext} from 'mattermost-redux/types/apps';
+
+import * as mattermost from '../mattermost/client';
+import * as zendesk from '../zendesk/client';
+
+import store from './store';
+import {getTicketForPost} from './model';
+
+const username = process.env.ZENDESK_USERNAME as string;
+const zdToken = process.env.ZENDESK_API_TOKEN as string;
+const apiURL = process.env.ZENDESK_URL + '/api/v2' as string;
+
+class App {
+    async createTicketFromPost(appCall: AppCall): Promise<string> {
+        const ticket = getTicketForPost(appCall.values);
+
+        const zdClient = zendesk.newClient(username, zdToken, apiURL);
+
+        const result = await zdClient.tickets.create(ticket);
```

Comment:
See what type is returned from the client and rename accordingly.   Intention revealing name needed